airflow/operators/python.py

```diff
@@ -249,6 +250,9 @@ def execute(self, context: Dict):
                                            f'multiple_outputs, found {key} ({type(key)}) instead')
             for key, value in return_value.items():
                 self.xcom_push(context, key, value)
+        elif is_container(return_value):
+            for i, value in enumerate(return_value):
+                self.xcom_push(context, str(i), value)
```

Comment:
```suggestion
                self.xcom_push(context, f"return_value{i}", value)
```
How about something more informative?

Reply:
gotcha fixed that, but wouldn't you want to index xcomarg though? like `output[0]` vs `output["return_value_0"]`

Reply:
But when you think about it anywhere you have to index each item to pass to another task is better to take a generic iterable instead and just pass entire container output instead of each individual item so I see your point